packages/react/src/dialog/Dialog.css.ts

```diff
@@ -0,0 +1,65 @@
+import { theme } from "../styles";
+import { type RecipeVariants, recipe, style } from "../vanilla-extract";
+
+export const content = recipe({
+  base: [
+    {
+      bg: "white",
+      rounded: "lg",
+      shadow: "md",
+    },
+    style({
+      position: "relative",
+    }),
+  ],
+  variants: {
+    size: {
+      sm: style({
+        maxWidth: "90dvw",
```

Comment:
we can move this to the `base` style since the maxWidth is same for all 3 variants